types/vimeo__player/index.d.ts

```diff
@@ -86,6 +87,9 @@ export interface ChapterChangeEvent {
     index: number;
 }
 
+// tslint:disable-next-line:no-empty-interface
+export interface ChapterChangeEvent extends VimeoChapter {}
```

Comment:
Since `VimeoChapter` and the `ChapterChangeEvent` have the same structure, I opted to extend here instead of passing `VimeoChapter` in the `EventMap` in order to maintain the current style of `XXXEvent`.

Some other options:
- Just pass `VimeoChapter` directly within the `EventMap`
- Redeclare `VimeoChapter` with the exact same shape as `ChapterChangeEvent`

The same is true for `CameraChangeEvent` below.

If there are no issues with this approach, I'll leave it, but definitely open to suggestions.